storybookjs/frontpage
# Highlight current page in sidebar, and other fixes

Will also automatically open a sidebar accordion that contains the current page

### How to test

1. Open the [deploy preview](https://deploy-preview-630--storybook-frontpage.netlify.app/docs)
2. Confirm the "Install" page is highlighted in the sidebar
3. Navigate to a level 3 page (like [End-to-end testing](https://deploy-preview-630--storybook-frontpage.netlify.app/docs/writing-tests/stories-in-end-to-end-tests))
4. If you navigated by clicking a sidebar link, refresh the page
5. Confirm that the current page's accordion menu is open and the current page is highlighted
